exercises/count/exercise.js

```diff
@@ -15,7 +15,7 @@ var db, url = 'mongodb://localhost:27017/learnyoumongo'
 exercise.addSetup(function(mode, cb) {
   var self = this
   var numbers = [1, 3, 10]
-  var number = numbers[Math.floor(Math.random() * numbers.length)]
+  var number = numbers[Math.floor(Math.random() * (numbers.length-1))]
```

Comment:
Right. So,

[3,10] > [1] = 2
[10] > [3] = 1
[] > [10] = 0

I'm cutting out 10 so that we never try to check for ages larger than 10, preventing a result of 0.

We could probably accomplish the same thing by doing:

```js
var number = numbers[Math.floor(Math.random() * numbers.length)] - 1
```

Which would give us [0,3,9] keeping 3 values. Then count could be either 1, 2, or 3.